VetClinicApi.Database.Migrations/Migrations/Migration_002_CreateAnimalTable.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FluentMigrator;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VetClinicApi.Database.Migrations.Migrations;
+
+[Migration(MigrationVersioning.CreateAnimalTable)]
+public class Migration_002_CreateAnimalTable : Migration
+{
+    public override void Up()
+    {
+        Create.Table("T_Animal")
+            .WithColumn("Id").AsInt32().PrimaryKey().Identity()
+            .WithColumn("CustomerId").AsInt32().NotNullable()
+            .WithColumn("AnimalTypeId").AsInt32().NotNullable()
+            .WithColumn("Name").AsString().NotNullable()
+            .WithColumn("Breed").AsString().NotNullable()
+            .WithColumn("BirthDate").AsDate().NotNullable()
+            .WithColumn("IsVaccinate").AsBoolean().NotNullable()
+            .WithColumn("RegistrationDate").AsDateTime().NotNullable()
+            .WithColumn("LastEditDate").AsDateTime().NotNullable();
+        Create.Table("T_AnimalType")
```

Comment:
Move animal type into separated migration. AnimalType table should be 2nd migration and Animal table with both FK should be 3rd migration